bin/deploy

```diff
@@ -14,26 +14,26 @@ deploy_dynamic() {
   helm upgrade $RELEASE_NAME ./helm_deploy/laa-estimate-eligibility/. \
                 --install --wait \
                 --namespace=${K8S_NAMESPACE} \
-                --values ./helm_deploy/laa-estimate-eligibility/values/$NAMESPACE.yaml \
+                --values ./helm_deploy/laa-estimate-eligibility/values/uat.yaml \
                 --set image.repository="$ECR_ENDPOINT" \
-                --set image.tag="$CIRCLE_SHA1" \
+                --set image.tag="uat-$CIRCLE_SHA1" \
                 --set ingress.annotations."external-dns\.alpha\.kubernetes\.io/set-identifier"="$IDENTIFIER" \
                 --set ingress.hosts="{$RELEASE_HOST}" \
                 --set deploy.host="$RELEASE_HOST"
 }
 
-deploy_static() {
+deploy_standard() {
   helm upgrade check-client-qualifies ./helm_deploy/laa-estimate-eligibility/. \
                           --install --wait \
                           --namespace=${K8S_NAMESPACE} \
                           --values ./helm_deploy/laa-estimate-eligibility/values/$NAMESPACE.yaml \
                           --set image.repository="$ECR_ENDPOINT" \
-                          --set image.tag="${CIRCLE_SHA1}"
+                          --set image.tag="production-$CIRCLE_SHA1"
 }
 
 
-if [[ "$NAMESPACE" == "uat" ]]; then
-  deploy_dynamic
+if [[ "$NAMESPACE" == "branch" ]]; then
```

Comment:
I think "namespace" becomes a confusing variable name if we change its usage to be directly the deployment type and not [analogous to] the k8s namespace

Reply:
You're right, I'm overloading this variable. Will fix.